### DISCUSSION TITLE
Welcome to tontine Discussions!

### DISCUSSION THREAD

**Discussion by feuzeu** [lagdo/tontine#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Zohaibahmed82** [comment]
Hello

**Comment by Zohaibahmed82** [comment]
In this repository some code is missing in side menu like tours etc

**Reply by feuzeu** [reply]
Hi, there has been some changes on the side menu lately.
The tours (or rounds) menu is now located under the Calendar menu entry.

**Reply by Zohaibahmed82** [reply]
I want to access old code how I can access

**Reply by Zohaibahmed82** [reply]
Including subscription tours etc according to YouTube video

**Reply by feuzeu** [reply]
Browse the v3x branch.
https://github.com/lagdo/tontine/tree/v3x

**Reply by feuzeu** [reply]
Be aware that I not sure to be maintaining that branch in the future.
So if you are using the software, make sure to migrate to the latest version. You can post an issue in case you have any.

**Comment by ZohaibAhmeddev** [comment]
Chrome process stopped before startup completed. Additional info: The system cannot find the path specified.
why give me this issue kindly guide me
this issue on cPanel what i can do

**Comment by feuzeu** [comment]
First make sure the headless Chrome binary is installed, then set the `CHROME_PATH` var to the proper value in your `.env` file.
If you still have issue, checkout the [chrome-php/chrome](https://github.com/chrome-php/chrome) package documentation to see how to make it work on your system.
